Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -169,22 +171,33 @@ public static async Task<(int totalCount, int newCount)> ImportAccountAsync(Func
         private static async Task<List<ImportItem>> scanAccountsAsync(Func<Account, Task<ApiExtended>> apiExtendedfunc, Account[] accounts, LibraryOptions libraryOptions)
         {
             var tasks = new List<Task<List<ImportItem>>>();
-            foreach (var account in accounts)
+
+            LogArchiver archiver = default;
+
+            if (Log.IsEnabled(Serilog.Events.LogEventLevel.Verbose))
+            {
+				var logfile = System.IO.Path.Combine(Configuration.Instance.LibationFiles, "LibraryScans.zip");
```

Comment:
Am I right that this means that there will be at most 1 stored zip file? ie: only one named LibraryScans.zip

Reply:
Correct. We just add more jsons into the same zip file so they're all in one place.